setup.py

```diff
@@ -27,6 +27,7 @@
     'click>=7,<8', 'requests>=2,<3',
     'pymongo>=3.10,<4', 'dnspython>=1.16,<2',
     'pyyaml>=5,<6', 'tabulate>=0.8,<0.9',
+    'numpy',
```

Comment:
This needs to be version limited - `numpy<2`

Reply:
Done.